demos/Grokking_Demo.ipynb

```diff
@@ -3274,7 +3292,7 @@
     "get_metrics(model, metric_cache, get_cos_sim, \"cos_sim\")\n",
```

Comment:
FTR this bugs out on the free Colab GPU:
```
OutOfMemoryError: CUDA out of memory. Tried to allocate 76.00 MiB. GPU 0 has a total capacty of 14.75 GiB of which 26.81 MiB is free. Process 2651 has 14.72 GiB memory in use. Of the allocated memory 14.05 GiB is allocated by PyTorch, and 512.72 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF
```